sql/core/src/test/resources/sql-tests/inputs/linear-regression.sql

```diff
@@ -0,0 +1,51 @@
+-- Test aggregate operator with codegen on and off.
```

Comment:
We don't need to do this, as we are testing functions here, not the Aggregate operator.